src/status_im2/contexts/chat/lightbox/utils.cljs

```diff
@@ -142,20 +157,31 @@
    :set-full-height? (reagent/atom false)
    :overlay-z-index  (reagent/atom 0)})
 
+(defn initialize-opacity
+  [size selected-index]
+  (vec (map-indexed
+        (fn [i _]
+          (if (= i selected-index)
+            (anim/use-val 1)
+            (anim/use-val 0)))
+        (range size))))
```

Comment:
`(range size)` will return a coll of numbers of size `(count size)`, but we are ignoring the numbers in the `map-indexed` function, so we could just use `size`.

```clojure
(vec
   (map-indexed (fn [i _]
                  (if (= i selected-index)
                    (anim/use-val 1)
                    (anim/use-val 0)))
                size))
```

but maybe this is not the best use for `map-indexed` :thinking:, since we are just iterating and we need a vector, what about `mapv`?

```clojure
(mapv #(if (= % selected-index)
           (anim/use-val 1)
           (anim/use-val 0))
        (range size))
```

Reply:
Thanks @ulisesmac :)